### PR TITLE
schedule: add metrcis for region scatter

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### What problem does this PR solve?

Add metrics in region scatter

image

### Release note

* No release note

### CODE REVIEW

File: server/schedule/region_scatterer.go [hunk 1]
```diff
@@ -137,6 +137,7 @@ const maxRetryLimit = 30
 func (r *RegionScatterer) ScatterRegionsByRange(startKey, endKey []byte, group string, retryLimit int) ([]*operator.Operator, map[uint64]error, error) {
 	regions := r.cluster.ScanRegions(startKey, endKey, -1)
 	if len(regions) < 1 {
+		scatterCounter.WithLabelValues("skip", "empty_region").Inc()
```
Comment (rleungx) [inline server/schedule/region_scatterer.go:140]: How about using `-` instead of `_`

**Comment by rleungx** [comment]
BTW, can we add the related information of the filter?

File: server/schedule/filter/filters.go [hunk 2]
```diff
@@ -759,6 +759,32 @@ func (f *isolationFilter) Target(opt *config.PersistOptions, store *core.StoreIn
 	return true
 }
 
+// NewStoreBusyFilter creates storeBusyFilter
+func NewStoreBusyFilter(scope string) *storeBusyFilter {
+	return &storeBusyFilter{scope: scope}
+}
+
+// StoreBusyFilter
+type storeBusyFilter struct {
```
Comment (rleungx) [inline server/schedule/filter/filters.go:768]: How about using StoreStateFilter

Comment (Yisaer) [reply server/schedule/filter/filters.go]: updated.

**Review by lhy1024 — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

File: metrics/grafana/pd.json [hunk 3]
```diff
@@ -6971,6 +6971,220 @@
...
+              "show": true
+            }
+          ],
+          "yaxis": {
+            "align": false,
+            "alignLevel": null
+          }
+        }
+      ],
+      "title": "Scatter and Splitter",
```
Comment (rleungx) [inline metrics/grafana/pd.json:7185]: Why is it called `Scatter and Splitter`? It seems there is nothing related to split?

**Pull request merged**